src/Servers/Kestrel/Core/src/Internal/AddressBinder.cs

```diff
@@ -151,21 +151,8 @@ public async Task BindAsync(AddressBindContext context, CancellationToken cancel
             context.ServerOptions.ApplyEndpointDefaults(httpDefault);
             await httpDefault.BindAsync(context, cancellationToken).ConfigureAwait(false);
 
-            // Conditional https default, only if a cert is available
-            var httpsDefault = ParseAddress(Constants.DefaultServerHttpsAddress, out _);
-            context.ServerOptions.ApplyEndpointDefaults(httpsDefault);
-
-            if (httpsDefault.IsTls || httpsDefault.TryUseHttps())
-            {
-                await httpsDefault.BindAsync(context, cancellationToken).ConfigureAwait(false);
-                context.Logger.LogDebug(CoreStrings.BindingToDefaultAddresses,
```

Comment:
https://github.com/dotnet/aspnetcore/blob/d96a100bddc72606f7417b665428411388b8ac54/src/Servers/Kestrel/Core/src/CoreStrings.resx#L426
This message is probably not used anymore and can be removed.